resources/schema.py

```diff
@@ -70,17 +112,16 @@ class Meta:
             "lighting",
             "suitable_boat_types",
         ]
-        geojson_field = "location"
-        interfaces = (relay.Node,)
 
 
 class BerthTypeNode(DjangoObjectType):
+    width = graphene.Float(description=_("width (m)"))
```

Comment:
please, keep the fields where they were and keep them `required`. as I said, there is no uniformity across apps on whether fields come before `Meta` or after in the GQL type classes, but within each `schema.py` file there is such uniformity, and this file has them after the `Meta` 🙂

Reply:
I totally forgot about this one 😅 I ditched all the `tmp/` commits and I think it went down the sink with them.